darwin/DarwinProcess.c

```diff
@@ -39,7 +39,7 @@ const ProcessFieldData Process_fields[LAST_PROCESSFIELD] = {
    [M_VIRT] = { .name = "M_VIRT", .title = " VIRT ", .description = "Total program size in virtual memory", .flags = 0, .defaultSortDesc = true, },
    [M_RESIDENT] = { .name = "M_RESIDENT", .title = "  RES ", .description = "Resident set size, size of the text and data sections, plus stack usage", .flags = 0, .defaultSortDesc = true, },
    [ST_UID] = { .name = "ST_UID", .title = "UID", .description = "User ID of the process owner", .flags = 0, },
-   [PERCENT_CPU] = { .name = "PERCENT_CPU", .title = "CPU% ", .description = "Percentage of the CPU time the process used in the last sampling", .flags = 0, .defaultSortDesc = true, .autoWidth = true, },
+   [PERCENT_CPU] = { .name = "PERCENT_CPU", .title = " CPU%", .description = "Percentage of the CPU time the process used in the last sampling", .flags = 0, .defaultSortDesc = true, .autoWidth = true, },
```

Comment:
Your next commit changes the same line. You should squash the two changes of the same line into one.

Reply:
Actually, I want to split `CPU%` and `NCPU%` column related changes in different commits. And those change in white space position between `CPU%` and `NCPU%` column is required to maintain alignment between title and their values for the respective commits. For example:

- If we would remove keep `.title = "CPU%"` and `.title = "NCPU%"` in commit https://github.com/htop-dev/htop/pull/959/commits/9d363d9cb34485db542eb2974570ad8511284a62, we'll get misaligned columns.
![CPU%-u](https://user-images.githubusercontent.com/28824347/163539855-9dd2c836-68ce-450b-aa4f-d0b06c7b2e23.png)
![CPU%-2-u](https://user-images.githubusercontent.com/28824347/163539856-a856cd6e-e697-4f21-924f-d7208800276c.png)

- Same things happens when keep title like this: `.title = "CPU%"` and `.title = "NPCU% "`.
![NCPU%-u ](https://user-images.githubusercontent.com/28824347/163539985-6c1df961-ad8c-41e2-9034-17e262a2d69d.png)

So having same line change in two different commit allow me keep title aligned for respective commits.

Reply:
@etanot I know you intended the "CPU%" and "NCPU%" column changes to reside in different commits. What I mean is, in 20fe4bc283f4f8c48eaa892f9250120696019061, you changed the column title of "CPU%" again. That part of the changes should be merged to 9d363d9cb34485db542eb2974570ad8511284a62, so that 20fe4bc283f4f8c48eaa892f9250120696019061 contains only "NCPU%" column change and no "CPU%" column change.

Reply:
> https://github.com/htop-dev/htop/commit/20fe4bc283f4f8c48eaa892f9250120696019061 contains only "NCPU%" column change and no "CPU%" column change.

Reason for not doing so is that, commit 9d363d9 would have misaligned title with respect to their values. To reproduce this follow following steps:

- Clone this branch.
- `git checkout 9d363d9`.
- Apply [patch.txt](https://github.com/htop-dev/htop/files/8500983/patch.txt) patch (`git apply patch.txt`)
- Compile htop and run htop.
- Now run a CPU intensive process. For example, run `ffmpeg` command on a video (`ffmpeg -i in.mp4 out.mp4`).

We'll get an output something like this:
![misaligned](https://user-images.githubusercontent.com/28824347/163700553-0c3f6a8d-c9e9-4220-ba0d-eafc16874aa7.png)
MEM% column's values are far away from column title.

My question is that, **is it okay to have a commit in a pull request which breaks the alignment for that specific commit, but overall pull request doesn't have this problem?**

Reply:
When @BenBE pushed a new commit, this change I suggested to @etanot seems to be reverted.

There's no need for the beginning padding space for the "CPU%" column, since the CPU% column title is now printed with right alignment.